### DISCUSSION TITLE
Is this integration for Mitsubishi Heavy Industries (MHI) or Mitsubishi Electric airco's?

### DISCUSSION THREAD

**Discussion by jan-gerard** [gysmo38/mitsubishi2MQTT#261] [General]
Hi,
I fail to find whether this integration is intended for Mitsubishi Heavy Industries (MHI) or Mitsubishi Electric airco's. Those are essentially different, although the naming is indeed confusing. But those are different companies making different, incompatible, products.

**Comment by vperas** [comment]
Dont know if it works on an MHI unit; it working on mine MSZ-HJ35VA from Mitsubishi Electric

EDIT: This guy has made one for MHI: https://rjdekker.github.io/MHI2MQTT/
